### DISCUSSION TITLE
New mechanics for Yeti

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2794] [Ideas]
Here a two ideas for yeti

1. Throw more variety of enemies in addition to jumping snowball

2. A buttjump that makes all the stalactites fall off

If anyone is interested in the idea, comment on your opinion

**Comment by Supertux395** [comment]
We can instead make custom balls of snow thrown from yeti and snow particles on tux if he gets hit an animation of snow on his head and he will fall.
